pkg/controllers/podgroup/pg_controller_handler.go

```diff
@@ -93,6 +94,10 @@ func (pg *pgcontroller) addReplicaSet(obj interface{}) {
 		if podList != nil && len(podList.Items) > 0 {
 			pod := podList.Items[0]
 			klog.V(4).Infof("Try to create podgroup for pod %s/%s", pod.Namespace, pod.Name)
+			if !slices.Contains(pg.schedulerNames, pod.Spec.SchedulerName) {
+				klog.V(4).Infof("pod %s/%s field SchedulerName is not matched", pod.Namespace, pod.Name)
```

Comment:
Please change pod to upper case.

Reply:
And we can use `klog.Kobj` to print pod.namespace and pos.name.

Reply:
done